curvefs/src/metaserver/metaserver.cpp

```diff
@@ -222,6 +256,8 @@ void Metaserver::Stop() {
 
     LOG(INFO) << "MetaServer is going to quit";
 
+    externalServer_->Stop(0);
```

Comment:
you should check whether external ip is enabled otherwise, externalServer_ is null, this can cause segment fault.

Reply:
> you should check whether external ip is enabled otherwise, externalServer_ is null, this can cause segment fault.

fixed